boards/nrf_dk_common.dtsi

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2024 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		/* click-i2c alias is currently needed by libostentus */
+		click-i2c = &arduino_i2c;
+	};
+};
```

Comment:
The only reason why this file is here is in case we want to test Ostentus with the nRF DK boards, otherwise we can just delete this file.